pkg/commands/policy/policy.go

```diff
@@ -127,13 +127,16 @@ func (c *PolicyCommand) Process(ctx context.Context) error {
 
 			merr = errors.Join(merr, fmt.Errorf("failed: \"%s\" - %s", k, m.Message))
 		}
+	}
 
-		logger.DebugContext(ctx, "found missing approvals",
-			"teams", teams,
-			"users", users,
-		)
+	if len(teams) == 0 && len(users) == 0 {
+		return nil
```

Comment:
Could you add a code comment here explaining why merr is ignored in this case?

Reply:
Thanks, we actually want to return `merr` and left a comment explaining why